public/css/styles.css

```diff
@@ -0,0 +1,673 @@
+  padding-bottom: 5em;
+}
+
+footer p {
+  margin-top: 2em;
+}
+
+footer a{
+  text-decoration: none;
+  color: #FF6124;
+}
+
+/* -- media queries --*/
+
+@media only screen and (max-width: 700px) {
+  div[class=carousel_portfolio] {
+    display: block;
+    background-image: url("./../imgs/portfolio-sm.jpg");
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 32em;
+    height: 16em;
+    max-width: 100%;
+  }
+  img[class="carousel-lrg"] {
+    display: none;
+  }
+}
+
+@media only screen and (min-width: 300px) and (max-width: 850px) {
+  object {
+    display: none;
+  }
+  .awesome-sm {
+    border-bottom: 3px solid #FF6124;
+  }
+  header > p {
+    margin-bottom: .3em;
+    padding-bottom: .3em;
+  }
+}
+@media only screen and (min-width: 850px) {
+  object {
+    display: inline;
+    width: 14.5em;
+    margin-left: -1.75em;
+    margin-top: -5.7em;
+  }
+}
+@media only screen and (max-width: 850px) {
+ .nav-li {
+  font-size: .75em;
+ }
+}
+
+@media only screen and (min-width: 700px), only screen and (min-width: 700px) and (orientation: landscape) {
+  body {
+    margin: 0 auto;
+  }
+
+  .contact-us form {
+    width: 38%;
+    height: 20em;
+    margin-right: 3em;
+
+  }
+  .message-button {
+    float: right;
+  }
+  .flex-item-1 {
+    width: 31%;
+  }
+  .flex-item-2 {
+    width: 31.5%;
+  }
+  p {
+    margin-left: 10%;
+    margin-right: 12%;
+  }
+  .column-right {
+    width: 32em;
+    margin-right: 16%;
+  }
+  .column-left {
+    margin-left: 16%;
+    width: 32em;
+  }
+
+}
+
+@media only screen and (max-width: 1200px) {
+  .flex-item-1 {
+    width: 35%;
+  }
+  .flex-item-2 {
+    width: 50%;
+  }
+   .flex-item-4 {
+    width: 50%;
+  }
+  .team-para {
+    max-width: 18em;
+    display: inline-flex;
+  }
+}
+
+@media only screen and (max-width: 1000px) {
+  .flex-item-1 {
+    width: 50%;
+  }
+}
+
+@media only screen and (max-width: 750px) {
+  .flex-item-2 {
+    width: 100%;
+  }
+  .client-wrapper {
+    display: block;
+  }
+  .selected:hover > .hidden-class {
+  display: block;
+  }
+}
+
+@media only screen and (max-width: 700px) {
+  .flex-item-1 {
+    width: 60%;
+  }
+  .flex-item-3 {
+    width: 33%;
+  }
+  .flex-item-4 {
+    width: 100%;
+  }
+  .client-wrapper {
+  display: block;
+  }
+  .nav-ul {
+    width: 100%;
+    display: flex;
+    display: -webkit-flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    align-content: center;
+    align-items: flex-start;
+    -webkit-justify-content: center;
+    -webkit-flex-direction: row;
+    -webkit-flex-wrap: wrap;
+  }
+  .nav-li {
+    font-size: 1em;
+    margin-right: 2em;
+    margin-bottom: .5em;
+    border: 2px solid black;
+    border-radius: .5em;
+    padding: .4em;
+  }
+}
+
+@media only screen and (max-width: 500px) {
+  .flex-item-3 {
+    width: 50%;
+    line-height: 3em;
+  }
+}
+
+@media only screen and (max-width: 1200px) {
+  .client-wrapper {
+    display: inline-flex;
+  }
+}
```

Comment:
This is a lot of media queries. Do you need all of them? Can you get by with 2-3? Is there something you could do to make your `flex-item-`s, in particular, more resilient and flexible?